### PR TITLE
Form section: Microverse Portfolio

### PR DESCRIPTION
**Pull request by Graycemuthui**
Created a contact section.
Created a form with input types and form validations.
Used styles.css to style the contact section.
Used proper GitHub flow.
Followed all project and general guidelines.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -252,41 +252,73 @@ <h3 class="connect-text">LET'S CONNECT</h3>
...
+    <!-- Contact Section -->
+    <section class="contact-section">
+      <div class="contact-container">
+        <h2 class="contact-header">Contact Me</h2>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form class="form-inputs">
```
Comment (m05e5) [inline index.html]: Hi Great job.
- [ ] As mentioned in the requirements, the form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html)

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @Graycemuthui , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] As mentioned in the requirements, the form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html)

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Graycemuthui** [comment]
Thank you :smile:

**Pull request merged**